feature/gnoi/system/tests/supervisor_switchover_test/README.md

```diff
@@ -6,12 +6,23 @@ Validate that the active supervisor can be switched.
 
 ## Procedure
 
-*   Issue gnoi.SwitchControlProcessor to the chassis with dual supervisor,
-    specifying the path to choose the standby RE/SUP.
-*   Ensure the SwitchControlProcessorResponse has the new active supervisor as
-    the one specified in the request.
-*   Validate the standby RE/SUP becomes the active after switchover
-*   Validate that all connected ports are re-enabled.
+*   Basic Switchover
+    *   Issue gnoi.SwitchControlProcessor to the chassis with dual supervisor,
+        specifying the path to choose the standby RE/SUP.
+    *   Ensure the SwitchControlProcessorResponse has the new active supervisor as
+        the one specified in the request.
+    *   Validate the standby RE/SUP becomes the active after switchover
+    *   Validate that all connected ports are re-enabled.
+
+*   Configuration consistent after switchover
+    *   Get the configuration before switchover.
+    *   Perform and validate switchover using the process outlined above.
+    *   Get configuration after switchover and validate that it matches the pre-switchover configuration.
+
+*  Configuration convergence after switchover
+    *   Perform and validate switchover using the process outlined above.
+    *   Push a large configuration
+    *   Get the configuration and validate that it is accepted whithin 60 seconds.
```

Comment:
I think these steps should be moved to gNMI-1.19?

I think this portion of the test is duplicate with gnmi-1.19?   The implementation should probably be moved here instead:

https://github.com/openconfig/featureprofiles/blob/64388db0ea85c18ed6e556324dc8f2dcc643ddfb/feature/platform/controllercard/tests/SetRequest_controll_card_switchover/README.md?plain=1#L1

Reply:
Ack. Will close this and open a new PR for gNMI-1.19.